README.md

```diff
@@ -1 +1,78 @@
-# libmulticonfig
\ No newline at end of file
+
+# Multi-configuration parser library using C
+
+`libmulticonfig` is a multi-configuration parser library which was implemented by C.
+The several configurations may be parsed by this library:
```

Comment:
s/The several configurations may be parsed by this library:/The configuration for the library is parsed in this order:/